deployment/artifacts/new-susbcription-mainnet.json

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
Should we rename to `marlin.json` or something analogous?

Reply:
Also, do we need `GlobalAllowList` in this file since it is already tracked in `mainnet.json`? I guess it is a remnant from the fact that the deployment included a fresh deployment of GlobalAllowList - but for the purposes of the subscription registry tracking for marlin, isn't really needed.